sbnobj/Common/POTAccounting/BNBSpillInfo.h

```diff
@@ -6,30 +6,30 @@
  * @file sbnobj/Common/POTAccounting/BNBSpillInfo.h
  * @author Joseph Zennamo, FNAL (jaz8600@fnal.gov)
  */
-
 #include <vector>
 
 namespace sbn {
+
   class BNBSpillInfo {
   public:
-    //typedef enum triggertype { kBNB, kEXT, kUnbiased, kNUMI } TriggerType; //!< Which beamline are we looking at
-    
+
+    unsigned int event;
+
     unsigned long int spill_time_s; //!< The IFDB Beam Spill Time, unit sec
     unsigned long int spill_time_ns; //!< The IFDB Beam Spill Time, unit nsec
 
-    unsigned int event;
-        
-    //BeamQuaility 
-    //quality//BNB
-    
+    float FOM; // Figure of Merit for BNB
```

Comment:
Please make this a Doxygen documentation comment:
```suggestion
    float FOM; ///! Figure of Merit for BNB
```